miden-lib/asm/miden/contracts/wallets/basic.masm

```diff
@@ -53,9 +58,18 @@ export.send_asset
     # is because the VM stack has a minimum size of 16 elements, trying to push
     # elements after the call to `create_note` would increase the stack in
     # addition to the minimum 16 elements.
-    push.0.0 padw movdnw.3 padw movdnw.3 movdn.11 drop
-    # => [ASSET, tag, note_type, RECIPIENT, ZERO, ZERO, 0, ...]
+    push.0 movdn.7 padw movdnw.2 padw movdnw.2
+    # => [tag, aux, note_type, RECIPIENT, 0, ZERO, ZERO, ...]
 
     exec.tx::create_note
-    # => [note_ptr, ZERO, ZERO, 0, ...]
+    # => [note_ptr, 0, 0, 0, ZERO, ZERO, ZERO, ...]
```

Comment:
This doesn't seem right, should it be like this?

```
# => [note_ptr,  0, ZERO, ZERO, ...]
```

Reply:
indeed